pkgs/os-specific/linux/rtl88xxau-aircrack/default.nix

```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   name = "rtl88xxau-aircrack-${kernel.version}-${version}";
-  rev = "fc0194c1d90453bf4943089ca237159ef19a7374";
+  rev = "c0ce81745eb3471a639f0efd4d556975153c666e";
   version = "${builtins.substring 0 6 rev}";
```

Comment:
Do you by chance know why this is not using the unstable-XXXX-XX-XX format?

Reply:
No clue. No mention in the initial PR of why and the only thing to touch the file so far was a global change.